solutions/standard/variables.tf

```diff
@@ -128,7 +128,7 @@ variable "private_engine_enabled" {
 variable "private_engine_name" {
   type        = string
   description = "The name of the IAM Engine used to configure a Secrets Manager private certificate engine for an existing instance."
-  default     = "private-engine-sm"
+  default     = "private-engine-secrets-manager"
```

Comment:
Perhaps go with `private-cert-engine`. I think the variable should also be renamed to `private_cert_engine_name`